### PR TITLE
Globalvariables

### PR DESCRIPTION
**Pull request by samifa308**
changed var in line-height and font-family for p tags and moved over footer styles BING BONG

### CODE REVIEW

File: css/global.css [hunk 1]
```diff
@@ -264,8 +264,9 @@ h5,
 p {
 	font-size: var(--font16);
 	font-weight: var(--font-weight-regular);
-	line-height: var(--line-height20);
+	line-height: var(--line-height26);
```
Comment (samifa308) [inline css/global.css:264]: matching figma design

File: css/global.css [hunk 2]
```diff
@@ -264,8 +264,9 @@ h5,
...
 	color: var(--color-white);
+	font-family: var(--body-font);
```
Comment (samifa308) [inline css/global.css:266]: open sans

File: css/global.css [hunk 3]
```diff
@@ -497,8 +498,44 @@ Feel free to change the min/max-width values to whatever you need. Use variables
...
+		flex-direction: column;
+		align-items: start;
+	}
+	.text-input .button2 {
+		margin-left: 0px;
+	}
+	form p {
+		width: 100%;
+		padding-bottom: var(--spacing10);
+	}
```
Comment (samifa308) [inline css/global.css:693]: moved footer from a repeated media querie

File: css/global.css [hunk 4]
```diff
@@ -420,8 +421,7 @@ form {
 }
 /* positioning for body to stop scroll when modal is open */
 .fixed {
-	position: fixed;
-	width: 100%;
+	overflow: hidden;
```
Comment (samifa308) [inline css/global.css]: this fixes the modal from jumping on top of the screen

**Review by SolivanLau — APPROVED** [review]
Worked with Roo, Samantha, Miranda, and Kyla on reorganizing **global.css**! 
- footer has been moved over to a pre-existing 1024px mq
- re-ordered all mqs at the bottom of css file
- dealt with merge conflicts 
- removed duplicate **hidden** and fixed styles

**Review by maxharrisnet — APPROVED** [review]
Looks good to me 🚀

**Pull request merged**